### DISCUSSION TITLE
Error: git push -u origin master

### DISCUSSION THREAD

**Discussion by jermain119** [community/community#22688] [New to GitHub]
<p>here is a screen shot of the output<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/f/f606f7a6d75dc7c1c1cadadf8658326f3d1683c7.png" data-download-href="/uploads/short-url/z6snT4hwhduKLnVRWgrEU3ZpR5l.png?dl=1" title="gitremote" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/40341079/181072603-ae0bfc32-41f4-44c2-be69-ed927a2c98ed.png" alt="gitremote" data-base62-sha1="z6snT4hwhduKLnVRWgrEU3ZpR5l" width="690" height="108" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/f/f606f7a6d75dc7c1c1cadadf8658326f3d1683c7_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">gitremote</span><span class="informations">1437×227 25.7 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by airtower-luna** [comment]
<p>That means Github doesn’t accept your SSH public key for authentication. You might need to <a href="https://docs.github.com/en/github/authenticating-to-github/adding-a-new-ssh-key-to-your-github-account">add it to your account</a>.</p>

**Comment by jermain119** [comment]
<p>hi thanks luna but i added the ssh key to GitHub  but git hub commands for adding to repository getting  error if you find some time through the day and can help me set it up would appreciated ?</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>If you have more questions please directly ask here on the forum and include the relevant details, for anyone to answer. I’m happy to help when I can, but I’m not going to promise any personal support.</p>
